### ISSUE TITLE
Allow Accordion to be displayed as open by default

### ISSUE THREAD

**Issue by IncredibleMrTim** [ONSdigital/design-system#1803]
Currently we are unable to display the Accordion component as open by default on setting the `aria-expanded` attribute and the `ons-collapsable--open` class, the accordion opens and then closes again as the page loads.

```
   <div id="accordion" class="ons-accordion">
  <div id="accordion-1" class="ons-collapsible ons-js-collapsible ons-collapsible--accordion ons-collapsible--open" data-btn-close="Hide" data-group="accordion">
    <div class="ons-collapsible__heading ons-js-collapsible-heading" aria-expanded="true">
      <div class="ons-collapsible__controls">
        <h2 class="ons-collapsible__title">Total retail turnover</h2>
```

**Comment by boxadesign** [comment]
https://github.com/ONSdigital/design-system/pull/1815
